kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/GeneralErrorCorrectionAction.java

```diff
@@ -1,5 +1,7 @@
 package edu.arizona.kfs.fp.document.web.struts;
 
+import static org.kuali.kfs.sys.KFSConstants.Booleans.TRUE;

```

Comment:
It's a junk value, only meant to convey its presence. I thought about using a random value, to more clearly communicate that it being there is all we care about, but seemed overkill.